utils/rpms/daos.spec

```diff
@@ -34,8 +34,10 @@ BuildRequires: protobuf-c-devel
 BuildRequires: spdk-devel >= 20, spdk-devel < 21
 %if (0%{?rhel} >= 7)
 BuildRequires: libisa-l-devel
+BuildRequires: libisa-l_crypto-devel
 %else
 BuildRequires: libisal-devel
+BuildRequires: libisal_crypto-devel
```

Comment:
I think you need to update the revision number at the top of the file and add a log comment as well

Reply:
I see the changelog at the bottom. It includes the revision number (I increased it to 1.1.0-20), however, the version number at the top is just 1.1.0 which I don't think should be changed.